kf_lib_data_ingest/etl/configuration/log.py

```diff
@@ -10,6 +11,26 @@
 )
 
 
+def _log_level_dict():
+    """
+    Create log levels dict, where a key is the string representation of
+    log level and a value is the int representation of the level.
```

Comment:
loggin already provides this in logging._nameToLevel

Reply:
yea i missed that